app/views/shared/_button.html.erb

```diff
@@ -0,0 +1,19 @@
+<a
+  class="
+    button
+    <% if local_assigns.include?(:start) && start == true %>
```

Comment:
You could put `<% start ||= false %>` at the beginning of the partial to simplify the logic, same with href.